app/views/govuk_publishing_components/components/docs/radio.yml

```diff
@@ -134,6 +134,25 @@ examples:
         text: "Yes"
       - value: "no"
         text: "No"
+  with_custom_page_heading_sizes:
+    description: |
+      This option allows the question to be a page heading (within a H1 element) and an option to have
```

Comment:
I think this text is potentially confusing. The size can be set regardless of whether it is a page heading or not. Suggestion:

```
This allows the size of the legend to be changed. Valid options are s, m, l, xl, defaulting to m if no option is passed. 

If the is_page_heading option is true and heading_size is not set, the text size will be xl. 
```

Also the description here states the options are uppercase letters, when in fact if those are passed they are ignored, accepted values are lowercase.

Reply:
Thanks - updated